### PR TITLE
Databases week3

### PR DESCRIPTION
**Pull request by aladdin91**
this is my homework for db week 3

### CODE REVIEW

File: databases/week3/meal_sharing_DB.sql [hunk 1]
```diff
@@ -0,0 +1,218 @@
+create database meal_sharing_DB;
+
+use meal_sharing_DB;
+
+CREATE TABLE
+    Meal(
+        `id` INT(10) UNIQUE NOT NULL PRIMARY KEY,
```
Comment (dboyadzhiev) [inline databases/week3/meal_sharing_DB.sql:7]: `PRIMARY KEY` = `UNIQUE NOT NULL` :)
when you set a Primary Key it requires the field to have unique and not null values

Comment (dboyadzhiev) [inline databases/week3/meal_sharing_DB.sql:7]: you can use `auto_increment` option here

File: databases/week3/meal_sharing_queries.sql [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+--Get reservations for a specific meal sorted by created_date
+
+SELECT
+    title,
+    reservation.created_date
+FROM meal
+    JOIN reservation ON meal.id = reservation.id
+WHERE
+    title LIKE 'Chicken and Rice'
```
Comment (dboyadzhiev) [inline databases/week3/meal_sharing_queries.sql:192]: use 'LIKE' in cases, you have a pattern to search for and = in cases you know the exact value
(like is a match slower than =)

Comment (dboyadzhiev) [inline databases/week3/meal_sharing_queries.sql:192]:
```suggestion
    meal.title = 'Chicken and Rice'
```

File: databases/week3/meal_sharing_queries.sql [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    Meal.title,
+    Review.stars
+FROM Meal
+    JOIN Review ON Meal.id = Review.id
+WHERE Review.stars >= 6;
+
+--Get reservations for a specific meal sorted by created_date
+
+SELECT
+    title,
```
Comment (dboyadzhiev) [inline databases/week3/meal_sharing_queries.sql:187]:
```suggestion
    meal.title,
```

**Review by dboyadzhiev — APPROVED** [review]
Well done!

**Comment by aladdin91** [comment]
thanks a lot for the feedback i did modify it according to the feedback 🙏

**Pull request merged**